From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nick Wellnhofer <wellnhofer@aevum.de>
Date: Tue, 20 Sep 2022 14:52:01 +0200
Subject: [PATCH] cmake: Set SOVERSION

---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 01e5bd62..33c3270e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -220,6 +220,7 @@ set_target_properties(
 	POSITION_INDEPENDENT_CODE ON
 	PREFIX lib
 	VERSION ${PROJECT_VERSION}
+	SOVERSION ${LIBXSLT_MAJOR_VERSION}
 )
 
 if(MSVC)
@@ -307,6 +308,7 @@ set_target_properties(
 	POSITION_INDEPENDENT_CODE ON
 	PREFIX lib
 	VERSION ${LIBEXSLT_VERSION}
+	SOVERSION ${LIBEXSLT_MAJOR_VERSION}
 )
 
 if(MSVC)
